### PR TITLE
[ION-874] Resizable Columns Plugin

### PR DESCRIPTION
**Pull request by gitstart-firstresonance**
## Checklist

- [x] The new code matches the existing patterns and styles.
- [x] The stories still work (run `yarn storybook`).
- [x] The stories are updated when relevant: `stories` for plugins, `knobs` for options.

### CODE REVIEW

File: packages/slate-plugins/src/deserializers/deserialize-html/__tests__/deserializeHTMLElement/pre-without-child.spec.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /** @jsx jsx */
 
-import { SlatePlugin } from '@udecode/slate-plugins-core/src';
+import { SlatePlugin } from '@udecode/slate-plugins-core';
```
Comment (alantrrs) [inline packages/slate-plugins/src/deserializers/deserialize-html/__tests__/deserializeHTMLElement/pre-without-child.spec.tsx:3]: Why these changes?

**Review by alantrrs — COMMENTED** [review]
- Please rebase on top of fr-fork
- Let's make sure there are no changes in the structure of the files in this branch. All the setup should be taken care by `fr-fork` so that this branch can cleanly be pushed to the main repository if/when we're ready.
- Since we want to contribute this back, let's make sure the Tables plugin can continue to work without the resizing functionality. If this is too much effort, please let me know and we can instead try to develop it as a separate plugin.

**Comment by gitstart-firstresonance** [comment]
Closing this PR in favor of the one create in tchami
